UI/html/viewAll.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="../css/new.css">
+    <title>PMS</title>
+
+</head>
+
+<body>
+    
+    <header>
+        <div class="logo">
```

Comment:
can you make the logo a link that takes you back to the homepage?

Reply:
This is just a suggestion. It wouldve been better to have one css file for features you have on all pages eg. the nav bar and footer. With that you will be too organised and also be able to make changes in that css file that affects all nav bars and footers in all pages of the whole project